cmake/f3dConfig.cmake.in

```diff
@@ -1,9 +1,31 @@
+# When this file is run by CMake through the find_package command, the
+# following targets will exist:
+#
+#   f3d::f3d           Target that contains the F3D executable
+#
+#   f3d::libf3d        Target that contains the libf3d
+#
+# The following local variables will also be defined:
+#
+#   libf3d_INCLUDE_DIR        Path to the include directory of public libf3d class
```

Comment:
It's a bit weird to make a `find_package(f3d)` and having to rely on `libf3d_*` variables

Reply:
f3d_* it is then